test/rf_test.dart

```diff
@@ -159,4 +160,30 @@ void main() {
           throwsA(const TypeMatcher<RohdHclException>()));
     });
   });
+  test('gen simple rf', () async {
+    const dataWidth = 16;
+    const addrWidth = 4;
+
+    const numWr = 2;
+    const numRd = 2;
+
+    final clk = SimpleClockGenerator(10).clk;
+    final reset = Logic();
+
+    final wrPorts = [
+      for (var i = 0; i < numWr; i++)
+        DataPortInterface(dataWidth, addrWidth)..en.put(0)
+    ];
+    final rdPorts = [
+      for (var i = 0; i < numRd; i++)
+        DataPortInterface(dataWidth, addrWidth)..en.put(0)
+    ];
+
+    final rf = RegisterFile(clk, reset, wrPorts, rdPorts, numEntries: 20);
+
+    await rf.build();
+
+    final res = rf.generateSynth();
+    File('build/${rf.definitionName}.v').openWrite().write(res);
```

Comment:
Use `File.writeAsStringSync` so that it doesn't run asynchronously on a stream.  I ran this as an experiment and only some files got generated because of a race between the file write and the tests completing.